src/overlays/actors/ovl_En_Elf/z_en_elf.c

```diff
@@ -1083,7 +1083,11 @@ void func_80A0461C(EnElf* this, PlayState* play) {
     } else {
         arrowPointedActor = play->actorCtx.targetCtx.arrowPointedActor;
 
-        if ((player->stateFlags1 & PLAYER_STATE1_10) || ((YREG(15) & 0x10) && func_800BC56C(play, 2))) {
+        // `R_SCENE_CAM_TYPE` is not composed of bit flags, but a specific bit is being checked for.
+        // In particular, `& 0x10` will pass for either `SCENE_CAM_TYPE_SHOP`, `SCENE_CAM_TYPE_FIXED`, or
+        // `SCENE_CAM_TYPE_SHOOTING_GALLERY`. However, only `SCENE_CAM_TYPE_SHOP` can have `VIEWPOINT_PIVOT` set.
```

Comment:
my turn to give my 2 cents on this :smirk: 
```c
        // `R_SCENE_CAM_TYPE` is not a bit field, but this conditional checks for a specific bit.
        // This `& 0x10` check will pass for either `SCENE_CAM_TYPE_SHOP`, `SCENE_CAM_TYPE_FIXED`, or
        // `SCENE_CAM_TYPE_SHOOTING_GALLERY`.
        // However, of these three, only `SCENE_CAM_TYPE_SHOP` is used with `VIEWPOINT_PIVOT`,
        // so here the bit check is equivalent to checking for `SCENE_CAM_TYPE_SHOP`.
```

Note: `SCENE_CAM_TYPE_HOUSE` is also used with `VIEWPOINT_PIVOT` but since it's 0x20 rules out by the bitwise check